invoicing_test.go

```diff
@@ -0,0 +1,406 @@
+package paypal_test
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	"github.com/plutov/paypal/v4"
+	"github.com/stretchr/testify/assert"
+)
+
+// All test values are defined here
+var devTestClientID = "AXy9orp-CDaHhBZ9C78QHW2BKZpACgroqo85_NIOa9mIfJ9QnSVKzY-X_rivR_fTUUr6aLjcJsj6sDur"
```

Comment:
Ideally we don't have any keys in tests, where did you get them? Should we move them to env vars?

Reply:
I obtained these from the `integration_test.go` file. They seem to be used in `order_test.go` as well. Should we ideally be importing them as env vars?